Content.Server/Nyanotrasen/Shuttles/Components/FTLWhitelistComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Server.Shuttles.Components;
+/// <summary>
+//Allows a grid to ignore FTL mass limit. Implemented to allow functioning of Spaceships, large shuttle grids/maps without issues.
```

Comment:
```suggestion
/// Allows a grid to ignore FTL mass limit. Implemented to allow functioning of Spaceships, large shuttle grids/maps without issues.
```

Needs to be 3 slashes in order to be a documentation comment.